dataStore/modules/memory/memoryPullAndStore.py

```diff
@@ -0,0 +1,96 @@
+__author__ = "李旭升 <programmerli@foxmail.com>"
+__copyright__ = "Licensed under GPLv2 or later."
+
+from dataStore.lepdClient.LepdClient import LepdClient
+from dataStore.influxDbUtil.dbUtil import myInfluxDbClient
+
+import time
+
+'''
+用lepdClient请求数据并把返回的数据用influxDbClient存储到InfluxDB中
```

Comment:
remove all Chinese chars from the code, including your Chinese names,  LEP will go globally eventually, we need to get ready for that :)